cmake/modules/Findthrift.cmake

```diff
@@ -0,0 +1,56 @@
+# This module defines
+#  thrift_LIBRARIES, libraries to link
+#  thrift_INCLUDE_DIR, where to find thrift headers
+#  thrift_COMPILER, thrift compiler executable
+#  thrift_FOUND, If false, do not try to use ant
+
+# prefer the thrift version supplied in thrift_HOME (cmake -Dthrift_HOME then environment)
+find_path(thrift_INCLUDE_DIR
+    NAMES
```

Comment:
indent with two spaces, please.

Reply:
sure, updated all nits for Findthrift.cmake using formatted for cmake: https://github.com/cheshirekow/cmake_format